ch02_elf/04_segment/hello.c

```diff
@@ -0,0 +1,10 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <threads.h>
+
+thread_local uint64_t tls_variable = 0xabcdabcdabcdabcd;
```

Comment:
```suggestion

_Thread_local uint64_t tls_variable = 0xabcdabcdabcdabcd;
```

これ本文中だと `#include <threads.h>` がなくて `_Thread_local` キーワードですよね。

Reply:
そうです。ありがとうございます。